src/hooks/useGatewaysPerEpoch.ts

```diff
@@ -0,0 +1,37 @@
+import { useGlobalState } from '@src/store';
+import { useQuery } from '@tanstack/react-query';
+import useEpochs from './useEpochs';
+
+export type GatewayEpochCount = {
+  epochIndex: number;
+  totalEligibleGateways: number;
+};
+
+const useGatewaysPerEpoch = () => {
```

Comment:
wondering if it might be better to just add filters to the useGateways hook

Reply:
useGateways() gets us AoGateway objects, but this hook gives us number of gateways for each epoch. useGateways has no ability to read gateway objects in an epoch.